api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/DbEgressEventTargetProperty.java

```diff
@@ -0,0 +1,35 @@
+package org.pmiops.workbench.actionaudit.targetproperties;
+
+import java.util.function.Function;
+import org.jetbrains.annotations.NotNull;
+import org.pmiops.workbench.db.model.DbEgressEvent;
+
+/** Action properties relating to a stored high-egress event within the workbench. */
+public enum DbEgressEventTargetProperty implements ModelBackedTargetProperty<DbEgressEvent> {
+  EGRESS_EVENT_ID("egress_event_id", event -> Long.toString(event.getEgressEventId())),
+  USER_ID(
+      "user_id",
+      event -> event.getUser() != null ? Long.toString(event.getUser().getUserId()) : null),
+  CREATION_TIME("creation_time", PropertyUtils.stringOrNull(DbEgressEvent::getCreationTime)),
```

Comment:
`stringOrNull()` is a new function which is used many times in this PR.  It is meant to replace the Kotlin expression `foo?.toString()` which works in the same way Javascript/Typescript does: if `foo` is a valid entity object, call `toString()` on it, otherwise return null/undefined.

See the function `stringOrNull()` for more details.  Let me know if the comments there need improvement.